srcpkgs/monero-gui/template

```diff
@@ -60,7 +60,7 @@ post_extract() {
 	bsdtar xzf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/monero-${_monero_version}.tar.gz --strip-components 1 -C monero
 	bsdtar xzf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${_rapidjson_gitrev}.tar.gz --strip-components 1 -C monero/external/rapidjson
 	bsdtar xzf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${_supercop_gitrev}.tar.gz --strip-components 1 -C monero/external/supercop
-	bsdtar xzf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/v${_randomx_version}.tar.gz --strip-components 1 -C monero/external/randomx
+	bsdtar xzf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${_randomx_gitrev}.tar.gz --strip-components 1 -C monero/external/randomx
```

Comment:
I think this should work and it's shorter
```suggestion
	bsdtar xzf ${wrksrc}/${_randomx_gitrev}.tar.gz --strip-components 1 -C monero/external/randomx
```

Reply:
At post_extract, there are two locations where the tarballs are availble:

    /void-packages/hostdir/sources/monero-gui-0.18.1.2/rapidjson-129d19ba7f496df5e33658527a7158c79b99c21c.tar.gz
    /host/sources/monero-gui-0.18.1.2/rapidjson-129d19ba7f496df5e33658527a7158c79b99c21c.tar.gz

`$wrksrc` is `/builddir/monero-gui-0.18.1.2`

Reply:
Any argument against removing `skip_extract` and using `mv`'s instead of `bsdtar` in `post_extract`?

Reply:
> Any argument against removing `skip_extract` and using `mv`'s instead of `bsdtar` in `post_extract`?

I think there is no problem if you do just like you described.

Reply:
Manual explains that if there are multiple distfiles unpacked, each gets another level of directory, so the monero source would go to `/builddir/monero-gui-0.18.1.2/monero-gui-0.18.1.2` instead of `/builddir/monero-gui-0.18.1.2`. I actually tried the `mv` variant, but due to that difference in behavior, the rest of the template became ugly.

Reply:
@paper42 I reverted this to the original path. Let me know if there's a better I'm unaware of.